### DISCUSSION TITLE
2.8.0-rc.1

### DISCUSSION THREAD

**Discussion by ruibaby** [halo-dev/halo#4292] [General]
## 新特性

- 支持备份和恢复功能。 by @JohnNiang in <https://github.com/halo-dev/halo/pull/4206>
- 支持通过上传和剪裁的方式设置用户头像。 by @LIlGG in <https://github.com/halo-dev/halo/pull/4253>
- 支持静态资源映射。 by @JohnNiang in <https://github.com/halo-dev/halo/pull/4285>

## 功能优化

- 升级默认编辑器版本，支持设置字体颜色、大小等功能，优化使用体验。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4288>
- Console 端编辑文章时允许同类型的编辑器切换。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4180>
- 重构 Console 端数据列表的筛选项 UI，并提供全局的筛选列表组件和搜索输入框组件。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4182>
- 重构 Console 端评论数据列表的筛选项 UI 和逻辑。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4195>
- 重构 Console 端附件数据列表的筛选项 UI 和逻辑。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4194>
- 重构 Console 端文章数据列表的筛选项 UI 和逻辑。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4193>
- 重构 Console 端系统设置页面的路由结构。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4196>
- Console 端的文章管理列表支持在地址栏记录筛选条件。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4102>
- Console 端的附件管理列表支持在地址栏记录筛选条件。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4210>
- Console 端的评论管理列表支持在地址栏记录筛选条件。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4209>
- Console 端的页面管理列表支持在地址栏记录筛选条件。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4208>
- 优化去除 HTML 重复 Meta 标签的正则表达式。 by @guqing in <https://github.com/halo-dev/halo/pull/4235>
- 优化 Console 端对异步数据轮询的间隔时间。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4246>
- Console 端关于用户密码输入的部分添加基本的验证。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4257>
- 移除 Console 端用户编辑表单中用户名的校验规则，防止旧版本 Halo 升级之后无法修改资料。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4276>

## 问题修复

- 修复生产模式下插件安装时的位置信息为绝对路径会影响迁移的问题。 by @guqing in <https://github.com/halo-dev/halo/pull/4179>
- 修复插件被卸载时 delete 生命周期方法不会被触发的问题。 by @guqing in <https://github.com/halo-dev/halo/pull/4241>
- 修复由于多线程环境下导致的插件卸载时的路由异常问题。 by @guqing in <https://github.com/halo-dev/halo/pull/4245>
- 修复编辑文章时，原来的图片尺寸被自动修改的问题。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4264>

## 依赖更新

- 升级 Spring Boot 至 3.1.2。 by @JohnNiang in <https://github.com/halo-dev/halo/pull/4284>
- 升级预设插件版本。 by @JohnNiang in <https://github.com/halo-dev/halo/pull/4287>

## 开发者相关

- Console 端插件详情选项卡支持通过插件扩展。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4041>
- Console 端全局注册 AnnotationsForm 组件，以暴露给插件使用。 by @ruibaby in <https://github.com/halo-dev/halo/pull/4212>
- 优化评论扩展点的使用方式。 by @guqing in <https://github.com/halo-dev/halo/pull/4249>
- 为 Console 端的 FormKit 表单添加附件策略和分组的输入类型。 by @LIlGG in <https://github.com/halo-dev/halo/pull/4258>
- 提供用户名密码认证扩展。 by @JohnNiang in <https://github.com/halo-dev/halo/pull/4265>
- 按条件渲染评论组件以简化主题端对评论组件是否显示的条件控制。 by @guqing in <https://github.com/halo-dev/halo/pull/4271>
- Extension 查询接口支持排序参数。 by @JohnNiang in <https://github.com/halo-dev/halo/pull/4274>

**Full Changelog**: https://github.com/halo-dev/halo/compare/v2.7.0...v2.8.0-rc.1

<hr /><em>This discussion was created from the release <a href='https://github.com/halo-dev/halo/releases/tag/v2.8.0-rc.1'>2.8.0-rc.1</a>.</em>

**Comment by zhaoshucai** [comment]
什么时候可以支持Zblog迁移过来啊。

**Reply by ruibaby** [reply]
可以关注一下：https://github.com/halo-sigs/plugin-migrate/issues/2


@LIlGG 应该已经在实现结构调整了，后续其他平台的导入我们期望社区能够提供实现。
